packages/veritone-react-common/src/components/Parts/Buttons/PillButton/styles.scss

```diff
@@ -0,0 +1,42 @@
+@import 'src/styles/modules/variables';
+
+.pillButton {
+  border: 1px solid #e0e0e0;
+  border-radius: 20px;
+  padding: 10px;
+  font-size: 14px;
+  display: inline-block;
+  background-color: white;
+
+  &:hover {
+    cursor: pointer;
+  }
+
+  > div {
+    float: left;
```

Comment:
in modern HTML using float for positioning is a totally unnecessary hack -- you should be using flexbox instead.